SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -134,7 +135,17 @@ bool CreateGlobalStateManager(Worker_SnapshotOutputStream* OutputStream)
 	Components.Add(CreateDeploymentData());
 	Components.Add(CreateGSMShutdownData());
 	Components.Add(CreateStartupActorManagerData());
-	Components.Add(EntityAcl(SpatialConstants::UnrealServerPermission, ComponentWriteAcl).CreateEntityAclData());
+
+	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
+
+	WorkerRequirementSet ReadACL;
+	for (const FName& WorkerType : SpatialGDKSettings->WorkerTypes)
```

Comment:
What's the difference between `WorkerTypes` and `ServerWorkerTypes` in the SpatialGDKSettings?

Reply:
`WorkerTypes` exists in the SpatialSDKSettings on this branch and `ServerWorkerTypes` doesn't

This is just rebased on later feature/actor-groups, where is ServerWorkerTypes intro'd?